### PR TITLE
*: cii badge

### PR DESCRIPTION
**Pull request by cyphar**
This is a pull request including all changes necessary in order for `umoci` to qualify for a [Core Infrastructure Initiative](https://bestpractices.coreinfrastructure.org/) badge. Currently we pass effectively all of the requirements, however we will need to improve our documentation in the future.

https://bestpractices.coreinfrastructure.org/projects/1084

Signed-off-by: Aleksa Sarai <asarai@suse.de>

### CODE REVIEW

**Comment by cyphar** [comment]
LGTM.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+private instead -- and cases where there are not enough maintainers to fulfil
+this requirement).
+
+Each commit should be self-contained and minimal (and should build and pass the
+tests individually), and commit messages should follow the Linux kernel style
+of commit messages. For more information see [&sect; 2 and 3 of
+`submitting-patches.rst` from the Linux kernel source][lk-commit].
+
+In addition, all commits must include a `Signed-off-by:` line in their
+description. This indicates that you cerify [the following statement, known as
```
Comment (vrothberg) [inline CONTRIBUTING.md]: ^ s/cerify/certify/ ?

**Comment by vrothberg** [comment]
LGTM.

**Comment by cyphar** [comment]
LGTM

**Pull request merged**